src/setupTest.js

```diff
@@ -1,10 +1,19 @@
 /* eslint-disable import/no-extraneous-dependencies */
-
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
 
+
+// TODO: Once there are no more console errors in tests, uncomment the code below
+// const { error } = global.console;
```

Comment:
Is this worth uncommenting now or something we're tracking for subsequent work?

Reply:
I didn't end up having time to fix all the test errors, so there are still 9 tests that fail if I uncomment. I'm going to open a ticket for subsequent work and leave this here so I don't have to look it up again. ;) 
(I suspect some of the errors are there because they're tricky to get rid of; I'm *hoping* none of them are coming from third-party code.)